pkgs/applications/networking/ps2client/default.nix

```
@@ -0,0 +1,25 @@
+{ fetchFromGitHub, stdenv }:
```

Normally, stdenv is the first derivation and the fetch function the second, followed by the other parameters.

```suggestion
{ stdenv, fetchFromGitHub }:
```